lib/internal/fs/cp/cp-sync.js

```diff
@@ -81,15 +81,12 @@ function getStats(src, dest, opts) {
 
 function onFile(srcStat, destStat, src, dest, opts) {
   if (!destStat) return copyFile(srcStat, src, dest, opts);
-  return mayCopyFile(srcStat, src, dest, opts);
-}
 
-// TODO(@anonrig): Move this function to C++.
-function mayCopyFile(srcStat, src, dest, opts) {
   if (opts.force) {
-    unlinkSync(dest);
-    return copyFile(srcStat, src, dest, opts);
-  } else if (opts.errorOnExist) {
+    return fsBinding.cpSyncOverrideFile(src, dest, opts.mode, opts.preserveTimestamps);
```

Comment:
Can you add this function to typings folder? particularly fs.d.ts

Reply:
sure, sorry my bad :sweat::+1:

Reply:
No worries. I think we should find an automated way of enforcing this. (If you remember contributing guideline, there was a rule that asks contributors to implement those enforcements)

Reply:
yeah I was thinking the same... since the js files are not type checked this is quite easy to miss :thinking:

Reply:
shall I open an issue to track this?

Reply:
Yeah sounds good to me

Reply:
https://github.com/nodejs/node/issues/58162 :slightly_smiling_face: